### DISCUSSION TITLE
issue with config file trying to send from vector agent in windows to vector in linux

### DISCUSSION THREAD

**Discussion by umpa385** [vectordotdev/vector#14005] [Q&A] [answered]
Wanted to see what I was doing wrong on this config. I'm trying to move the raw evtx files from the windows host to the linux host (I tried doing this locally using the "file" type and it worked as expected). Now i'm trying to move it over via vector and I'm getting this error below. Any help is appreciated. I double checked the firewall rules and port 6000 and port 9000 are allowed. 

_source{component_kind="source" component_id=vector_windows_application_evtx component_type=vector component_name=vector_windows_application_evtx}: vector::sources::vector::v2: Source future failed. error=TCP bind failed: Cannot assign requested address (os error 99)_

Windows: 
[sources.vector_windows_application_evtx]
type = "file"
include = ["C:\\Windows\\System32\\winevt\\Logs\\Application.evtx"]
read_from = "beginning"

[sinks.inputs_application]
type = "vector"
inputs = ["vector_windows_application_evtx"]
address = "X.X.X.X:6000"
version="2"

#[sinks.inputs_application.encoding]
#codec =  "text" #changing this to json makes each field value its own json object

linux:
[sources.vector_windows_application_evtx]
type = "vector"
address = "X.X.X.X:9000"
version ="2"

[sinks.vector_windows_application]
type = "file"
inputs = ["vector_windows_application_evtx"]
path = "/tmp/vector_windows-%Y-%m-%d.log"

[sinks.vector_windows_application.encoding]
codec = "text"

**Comment by jszwedko** [comment, marked as answer]
Hi @umpa385 !

That error indicates that Vector cannot bind to the configured address. Have you verified that the IP you are using is bindable by Vector? You could also try binding to 0.0.0.0 which will bind to all interfaces.

**Reply by umpa385** [reply]
using 0.0.0.0 worked but I will try and narrow it down to the correct IP address where the vector data is coming from, and confirm that the port isn't being used by another service. Also I will share my windows workaround once I get it working.
